Commands/Properties/AssemblyInfo.cs

```diff
@@ -44,6 +44,6 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("2.20.1711.0")]
-[assembly: AssemblyFileVersion("2.20.1711.0")]
+[assembly: AssemblyVersion("2.20.1711.1")]
```

Comment:
Please do not increase the version number. We will maintain the version numbering.